packages/optimizer/README.md

```diff
@@ -83,7 +83,8 @@ You can find a sample implementation [here](demo/simple/). If you're using the e
 Currently the following options are supported:
 
 * **ampUrl**: an URL string pointing to the valid AMP version. Required by the AddAmpLink transformer.
-* **imageBasePath**: a base URL or path required for resolving an image file from a given image `src` attribute. Required by the AddBlurryImagePlaceholder transformer.
+* **blurredPlaceholders**: Enables blurry image placeholder generation. Default is `false`. Make sure to only enable  **Important:** blurry image placeholder computation is expensive. Make sure to only use it for static or cached pages.
```

Comment:
This sentence looks incomplete:  "make sure to only enable"

Reply:
Also, for consistency: s/Enable/enable

Reply:
done